tests/x11/wine.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test wine with a simple Windows program
+#   I decided for about the most simple and small Windows programs I could
+#   find because one of the things I dislike about the MS Windows ecoysystem
+#   is bloated applications.
+# Maintainer: Oliver Kurz <okurz@suse.de>
+
+use base 'x11test';
+use strict;
+use testapi;
+
+sub run {
+    select_console 'x11';
+    # Actually only wine should suffice for the tiny app but maybe it really
+    # requires .NET because on startup wine asks if it should install a
+    # wine-mono package
+    ensure_installed 'wine wine-mono', timeout => 360;
+    x11_start_program('xterm');
+    my $cmd = <<'EOF';
+wget http://keir.net/download/timer.zip
+unzip timer.zip
+EOF
+    assert_script_run($_) foreach (split /\n/, $cmd);
+    script_run 'wine timer.exe', 0;
+    assert_screen([qw(wine-timer wine-package-install-mono-cancel)], 600);
+    if (match_has_tag 'wine-package-install-mono-cancel') {
```

Comment:
You already install wine , why you need to cancel ? Looks like accepting bug as expected behavior

Reply:
good observation. For all recent tests I added I observed some bugs - this is why I wrote the tests because automation provides reliable reproduction :) It's https://bugzilla.suse.com/show_bug.cgi?id=1087860 and I planned to create soft-fail needles pointing to the bugs on production. I can also create the needles already on lord.arch and push them in the needle commits. I could also add a record_soft_failure here. All can work. Any preferences?

Reply:
make sense